### DISCUSSION TITLE
keyball39 ファームウェアが書き込めないです

### DISCUSSION THREAD

**Discussion by ebr1010** [Yowkees/keyball#720] [Q&A]
3-7．ProMicroの書き込みと動作確認まではできましたが、
remapでファームウェアが左右どちらとも途中までしか書き込めず、
以降は何度繰り返しても「互換性のあるデバイスが見つかりません」と表示され選択すらできなくなりました。
PCに接続すると「keyball39」とデバイスは認識していますが、
keyball39本体はLEDが赤く光り画面には文字は表示されません。
またピンセットでの文字入力もできません。
リセットボタンも機能していません。(そもそも正常に機能した場合どうなるかが分かりませんが…)

初期不良でしょうか？
ミスがあるとしたらなぜかパーツのハマりが悪く別部品で代用したり、なぜか刺さらない箇所がありますが…

解決方法をご教示いただけると幸いです。

**Comment by takashi700** [comment]
マイコンを表裏逆にはめてないでしょうか？
僕も同じことをしてしまいました。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。takashi700さんもご回答ありがとうございます。
takashi700さんのご指摘の不具合の可能性もあります。

ひとまず、ProMicroを基板から取り外し、ProMicro単体でUSB接続してデバイスとして認識されるかご確認をお願いします。
お手数ですがご確認お願いいたします。
